base/metrics/persistent_histogram_storage.cc

```diff
@@ -30,7 +30,8 @@ constexpr size_t kAllocSize = 1 << 20;  // 1 MiB
 void* AllocateLocalMemory(size_t size) {
   void* address;
 
-#if BUILDFLAG(IS_WIN)
+#if defined(STARBOARD)
```

Comment:
Long term is it okay to have a noop here? If we should clean it up, switch it to USE_HACKY_COBALT_CHANGES

Same thing with base/metrics/persistent_memory_allocator.cc